frontend/src/components/ProgressiveDeliveryComponent/ProgressiveDeliveryComponent.tsx

```diff
@@ -54,7 +56,7 @@ function simplifyManyToMany(edges: [string, string][]): [string, string][] {
   let index = 0;
   for (const [originsKey, count] of Object.entries(multi_origins)) {
       if (count > 1) {
-          const many_to_many_node = `soak-${index++}`;
+          const many_to_many_node = `${MANY_TO_MANY_NODE_LABEL}-${index++}`;
```

Comment:
Did you mean to leave the index part or did you want them all to be just "soak"?

Reply:
not sure I understand the question? this change does modify the behaviour, I'm just using a variable instead of hardcoding `soak`.

However, below the change implies that the box will be labeled `soak`, and not `soak-<idx>`. But we need to retain the index in the `id` so that they're actually separate nodes